server-spi-private/src/main/java/org/keycloak/policy/DigitsPasswordPolicyProviderFactory.java

```diff
@@ -26,7 +26,7 @@
  */
 public class DigitsPasswordPolicyProviderFactory implements PasswordPolicyProviderFactory {
 
-    static final String ID = "digits";
+    public static final String ID = "digits";
```

Comment:
Please explain why you are making this change.

Reply:
We currently got an assignment to create SCIM module that integrates into keycloak. And if our clients want to change the password of a user that does not fit the password policy we want to return the password policy within the error message within a json structure. 
So we want to parse the content from the password policy object within the current realm by using the corresponding constants.

I will rollback the unit tests on monday

Reply:
By the way I am fully aware that this is actually a violation of the purpose of the ID constants but these were the only constants wihtin the keycloak implementation that were matching the effective keys.